api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -39,6 +40,22 @@ message IPTagging {
     EXTERNAL = 2;
   }
 
+  // Describes how to apply the tags to the headers.
+  enum HeaderAction {
+    // (DEFAULT) The header specified in :ref:`ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`
+    // will be dropped, before the tags are applied. Sanitize will be run regardless of if the source is a trusted IP or not.
+    //
+    // Note that the header will be visible unsanitized to any filters downstream of the ip-tag-header filter, unless it has an *x-envoy* prefix.
+    SANITIZE = 0;
+
+    // Tags will be appended to the header specified in
+    // :ref:`ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`,
+    // leaving original values in place.
+    //
+    // Please note that this could cause the header to retain values set by the http client even if the client is from an untrusted IP.
+    APPEND_FORWARD = 1;
```

Comment:
What does "FORWARD" imply?

I'm assuming the goal is to have something similar to [APPEND_IF_EXISTS_OR_ADD](https://github.com/envoyproxy/envoy/blob/01eb316f00f8bd3c870ff6397668f6ea80cb7637/api/envoy/config/core/v3/base.proto#L410). Will it make sense to either use the same enum (probably not as the one suggested adds sanitization), or use the same enum name?